src/frontend/src/components/warnBox.ts

```diff
@@ -28,7 +30,10 @@ export const warnBox = ({
   }
   const contents: TemplateResult = html`
     <span class="c-card__label c-card__label--hasIcon" aria-hidden="true">
-      <i class="c-card__icon c-icon c-icon--error__flipped">${warningIcon}</i>
+      <i class="c-card__icon c-icon c-icon--error__flipped c-icon--inline"
+        >${warningIcon}</i
+      >
+      ${headerSlot}
```

Comment:
I assume that `html` function renders `undefined` well and it doesn't show anything weird in the html?

Reply:
Yes, then it is just skipped.